### PR TITLE
feat(*): using new plugin API

### PR DESCRIPTION
**Pull request by bpowell**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -15,24 +15,18 @@ repositories {
...
-	// https://mvnrepository.com/artifact/com.netflix.spinnaker.orca/orca-core
-	compile group: 'com.netflix.spinnaker.orca', name: 'orca-core', version: '7.6.2'
-
-	// https://mvnrepository.com/artifact/com.netflix.spinnaker.kork/kork-core
-	compile group: 'com.netflix.spinnaker.kork', name: 'kork-core', version: '3.12.2'
-
-	// https://mvnrepository.com/artifact/com.netflix.spinnaker.kork/kork-artifacts
-	compile group: 'com.netflix.spinnaker.kork', name: 'kork-artifacts', version: '3.12.2'
+	// https://mvnrepository.com/artifact/com.netflix.spinnaker.orca/orca-api
+	compile group: 'com.netflix.spinnaker.orca', name: 'orca-api', version: '1'
```
Comment (link108) [inline build.gradle]: version?

File: src/main/java/com/netflix/spinnaker/plugin/stage/HelloDataModel.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.netflix.spinnaker.plugin.stage;
+
+import lombok.Data;
+
+@Data
```
Comment (link108) [inline src/main/java/com/netflix/spinnaker/plugin/stage/HelloDataModel.java]: docs/comments

File: src/main/java/com/netflix/spinnaker/plugin/stage/HelloStage.java [hunk 3]
```diff
@@ -16,29 +16,25 @@
...
 import org.springframework.stereotype.Component;
 
-import javax.annotation.Nullable;
-
 import javax.annotation.PostConstruct;
+import java.util.HashMap;
+import java.util.Map;
 
 /**
  * Example stage that implements stage definition builder. By implementing StageDefinitionBuilder,
```
Comment (link108) [inline src/main/java/com/netflix/spinnaker/plugin/stage/HelloStage.java]: need to update this

File: src/main/java/com/netflix/spinnaker/plugin/stage/HelloStage.java [hunk 4]
```diff
@@ -55,33 +51,20 @@ public void init() {
...
-   * This method defines the set of tasks for this stage. This method is required.
-   *
-   * @param stage
-   * @param builder
-   */
   @Override
-  public void taskGraph(Stage stage, TaskNode.Builder builder) {
-    // Task name can be changed per stage,
-    builder.withTask(HelloTask.TASK_NAME, HelloTask.class);
+  public String getName() {
```
Comment (link108) [inline src/main/java/com/netflix/spinnaker/plugin/stage/HelloStage.java]: docs/comments

File: src/main/java/com/netflix/spinnaker/plugin/stage/HelloStage.java [hunk 5]
```diff
@@ -55,33 +51,20 @@ public void init() {
...
+    return "helloStage";
   }
 
-  /**
-   * Context for this stage. The contents of this context are shared with stages and tasks that execute after this stage.
-   */
-  public static final class HelloStageContext {
-    private final String yourName;
+  @Override
```
Comment (link108) [inline src/main/java/com/netflix/spinnaker/plugin/stage/HelloStage.java]: docs/comments

File: build.gradle (hunk 1 above)
Comment (bpowell) [reply build.gradle]: we don't have a version yet. maybe should add a comment saying "version number coming soon (tm)"?

Comment (link108) [reply build.gradle]: yup, I think a comment is a great idea

**Pull request merged**